package.json

```diff
@@ -19,6 +19,7 @@
     "storybook": "start-storybook -p 9001 -c ./scripts/.storybook",
     "storybook:build": "build-storybook -c ./scripts/.storybook -o dist-storybook --webpack-stats-json",
     "test": "jest",
```

Comment:
Not strictly related to this PR, but I find myself using watch mode a lot when developing test for forma.